### PR TITLE
Create elyagoubi.json

### PR DESCRIPTION
**Pull request by abdessattar23**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [ ] The file is in the `domains` folder and is in the JSON format.
- [ ] The file's name is all lowercased and alphanumeric. 
- [ ] You have completed your website. 
- [ ] The website is reachable.  
- [ ] You're not using Vercel.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [ ] There is sufficient information at the `owner` field.  

### https://abdessattar23.github.io
Website Screenshot

### CODE REVIEW

File: domains/elyagoubi.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "owner": {
+        "username": "abdessattar23",
+        "email": "elyagoubiabdessattar@gmail;com"
+    },
+    "record": {
+        "CNAME": "abdessattar23.github.io",
```
Comment (DEV-DIBSTER) [inline domains/elyagoubi.json:7]: Cannot have CNAME in conjunction with other records as per DNS rules.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

**Comment by abdessattar23** [comment]
@DEV-DIBSTER OK thanks for noticing that brother, I have fixed the records by changing the CNAME record into A record pointing on the IP Address

**Review by DEV-DIBSTER — DISMISSED** [review]

**Comment by DEV-DIBSTER** [comment]
Actually wait, you'll need to put them in an array as GitHub wants all of those IPs, there should be 4.

**Comment by abdessattar23** [comment]
@DEV-DIBSTER Ohhh I missed that too, sorry lmao can you check now if it is valid

**Review by DEV-DIBSTER — APPROVED** [review]

**Pull request merged**